### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by b1rdex**
https://github.com/electron/grunt-electron-installer/pull/95

### CODE REVIEW

**Comment by b1rdex** [comment]
@havenchyk dunno what you mean by `es2015 syntax`, it's already ok for me.

**Comment by b1rdex** [comment]
@havenchyk also I don't understand why pointed me here. This PR must be landed to both repos, because both of them have readme with squirrel events handling example. You can reopen prev PR and merge it.

**Comment by havenchyk** [comment]
@b1rdex you're right, sorry, I missed that it's already es2015. It's ok for me as well.

**Comment by havenchyk** [comment]
@b1rdex actually, grunt-electron-installer is just a grunt task right now, we can put all the info here and not to reopen request against grunt-electron-installer again.

File: README.md [hunk 1]
```diff
@@ -86,47 +86,70 @@ You should handle these events in your app's `main` entry point with something
...
       // we update to the new version - it's the opposite of
       // --squirrel-updated
+
       app.quit();
       return true;
   }
 };
-
-if (handleStartupEvent()) {
+if (handleSetupEvent()) {
```
Comment (havenchyk) [inline README.md]: Shouldn't it be at the top of `main.js`? @b1rdex

Comment (b1rdex) [inline README.md]: ack

**Comment by havenchyk** [comment]
Thanks @b1rdex!

**Pull request merged**